tests/atomics_test.c

```diff
@@ -13,14 +13,21 @@
  * permissions and limitations under the License.
  */
 
-#include <aws/common/atomics.h>
+#include <aws/common/atomics.inl>
 #include <aws/common/common.h>
 #include <aws/common/condition_variable.h>
 #include <aws/common/mutex.h>
 #include <aws/common/thread.h>
 
 #include <aws/testing/aws_test_harness.h>
 
+#ifdef _WIN32
+#    include <malloc.h>
+#    define alloca _alloca
```

Comment:
Beginning to wonder if we should just have a block of these in common.h, aws_alloca, aws_strncpy, etc. Gets around all the _s ridiculousness.